libraries/pyopaeuio/CMakeLists.txt

```diff
@@ -24,11 +24,19 @@
 ## ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 ## POSSIBILITY OF SUCH DAMAGE.
 
+set(PYOPAE_PYBIND11_VERSION "2.2.4")
+
 if (OPAE_WITH_PYBIND11)
     set(INCLUDE_DIRS "${OPAE_INCLUDE_PATH}:${pybind11_ROOT}/include")
     set(LINK_DIRS "${LIBRARY_OUTPUT_PATH}")
     set(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/timestamp)
 
+    set(SETUP_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/setup.py")
+    set(SETUP_PY    "${CMAKE_CURRENT_BINARY_DIR}/setup.py")
+
+    #configure_file(${SETUP_PY_IN} ${SETUP_PY})
+    configure_file(${SETUP_PY})
```

Comment:
This configure_file call will modify the file in place in the source directory.  Will that present an issue?

Reply:
Actually, this approach will dirty the source tree whenever a build is done.  We should probably avoid this.